### DISCUSSION TITLE
启动nonebot报错

### DISCUSSION THREAD

**Discussion by NealScx** [nonebot/discussions#368] [问答] [answered]
![1](https://user-images.githubusercontent.com/85844254/235322759-6ca4f09d-eb50-4d64-8a04-66049e4fb579.jpg)
![2](https://user-images.githubusercontent.com/85844254/235322754-7f256d35-cf29-4a31-8a1c-be492c295884.jpg)
![3](https://user-images.githubusercontent.com/85844254/235322752-6a5dbfbb-daec-4665-ba31-771a1ab26819.jpg)

**Comment by yanyongyu** [comment, marked as answer]
这个报错是petpet插件的问题，petpet插件现在已经合并至memes，请使用新版插件

**Reply by NealScx** [reply]
呜呜呜  正常运行了   谢谢大佬   弄一晚上也不知道是什么问题  
![u=1087187810,792969306 fm=253 fmt=auto app=138 f=JPEG](https://user-images.githubusercontent.com/85844254/235340221-72bfcba5-5fd1-4da6-bdfa-d9df4f1e7403.jpg)
